views/includes/inputs/fields/is-on-register-in-country-formed-input.html

```diff
@@ -1,29 +1,26 @@
+{% if title == "Tell us about the overseas entity" %}
+   {% set hintText = "Include the full name. For example, Jersey Financial Services Commission." %}
+{% else %}
+   {% set hintText = "Include its jurisdiction. For example, Hong Kong or State of Delaware." %}
+{% endif %}
+
 {% set entityPublicRegisterHtml %}
   {{ govukInput({
     errorMessage: errors.public_register_name if errors,
     id: "public_register_name",
     name: "public_register_name",
     value: public_register_name,
     hint: {
-      text: "Include the full name. For example, Jersey Financial Services Commission."
+      text: hintText
     },
     label: {
       text: "Name of register"
     }
   }) }}
 
-  {{ govukInput({
-    errorMessage: errors.public_register_jurisdiction if errors,
-    id: "public_register_jurisdiction",
-    name: "public_register_jurisdiction",
-    value: public_register_jurisdiction,
-    hint: {
-      text: "For example, Jersey."
-    },
-    label: {
-      text: "Jurisdiction"
-    }
-  }) }}
+  {% if title == "Tell us about the overseas entity" %}
```

Comment:
for this include, you could set a boolean in the including page to indicate whether or not to include Jurisdiction, maybe have an 'includeJurisdiction = true/false' set before calling the is-on-register-in-country-formed-input

Reply:
Probably you do not need to set it to false, but it makes it more readable on MOC and BOC